### PR TITLE
Added possibility to set a http network proxy

### PR DESCRIPTION
**Pull request by AndreasFetzer**
The QNetworkProxy will be set as proxy for the QNetworkAccessManager

### CODE REVIEW

File: qRestAPI.cpp [hunk 1]
```diff
@@ -88,6 +88,17 @@ void qRestAPIPrivate::init()
 }
 
 // --------------------------------------------------------------------------
+void qRestAPI::setHttpNetworkProxy(const QNetworkProxy &proxy)
+{
+  Q_D(qRestAPI);
+
+  if (d->NetworkManager != NULL)
+  {
```
Comment (jcfr) [inline qRestAPI.cpp]: Nitpicks: +2 space for consistent indent

**Comment by jcfr** [comment]
:+1:

**Comment by AndreasFetzer** [comment]
Thanks for your response. I changed the indent.

**Comment by jcfr** [comment]
Et voila.

**Pull request merged**